internal/kojiapi/api/openapi.yml

```diff
@@ -112,7 +112,7 @@ components:
           type: array
           items:
             $ref: '#/components/schemas/ImageStatus'
-        koji_task_id:
+        koji_build_id:
```

Comment:
@gicmo This change may interfere with `koji-osbuild`.

Reply:
FTR, indeed: https://github.com/osbuild/koji-osbuild/blob/f529ffe394f370e0ffde88d98f9ce038cbfd7937/plugins/builder/osbuild.py#L116

Reply:
I reverted this. Thanks @ondrejbudai, and sorry for dropping the ball on this!